### DISCUSSION TITLE
Have to refresh token often

### DISCUSSION THREAD

**Discussion by smcmahon67** [dgreif/ring#864] [Q&A]
I am running this under homebridge. It mostly seems to work. I check the homebridge log about once a day and notice that I have to rerun `ring-auth-cli`. Am I doing something wrong?

**Comment by dgreif** [comment]
Which platform are you running homebridge on?

**Reply by smcmahon67** [reply]
I'm running this on my Mac (macOS 12.2) at the moment. There was an update to this package after I raised this issue and I think it might be more stable now.

**Reply by smcmahon67** [reply]
Spoke too soon. I had to refresh the token again.
